finetuner/__init__.py

```diff
@@ -517,6 +519,29 @@ def build_model(
         )
 
 
+def download_huggingface_model(model_name: str, token: Optional[str] = None) -> str:
```

Comment:
is this meant to be public? if not, use an `_` in front of the name

Reply:
I was not sure, maybe private is better